src/cephadm/cephadm

```diff
@@ -2008,6 +2024,10 @@ def deploy_daemon_units(fsid, uid, gid, daemon_type, daemon_id, c,
             poststop = nfs_ganesha.get_rados_grace_container('remove')
             f.write(' '.join(poststop.run_cmd()) + '\n')
         elif daemon_type == CephIscsi.daemon_type:
+            # make sure we also stop the tcmu container
+            ceph_iscsi = CephIscsi.init(fsid, daemon_id)
+            tcmu_container = ceph_iscsi.get_tcmu_runner_container()
+            f.write('! '+ ' '.join(tcmu_container.stop_cmd()) + '\n')
```

Comment:
And we can stop it when we stop the main container

Reply:
ok. now we're in the realm of Kubernetes PODs. 

Would be great to have a small abstraction that would enable us group multiple containers in one systemd unit:

* OSDS need special starting containers
* NFS has the grace container
* ISCSI has tcmu-runner.

anyway. lgtm is it is right now.